apps/examples/src/examples.tsx

```diff
@@ -13,7 +13,7 @@ export type Example = {
 	loadComponent: () => Promise<ComponentType>
 }
 
-type Category = 'basic' | 'editor-api' | 'ui' | 'collaboration' | 'data/assets' | 'shapes/tools'
+type Category = 'basic' | 'editor-api' | 'ui' | 'collaboration' | 'data' | 'shapes-tools'
```

Comment:
this is a bit annoying but you'll need to update `generateExamplesContent` to match these categories